docs/installation/local.md

```diff
@@ -4,4 +4,46 @@ sidebar_position: 1
 
 # Local Installation
 
-For development purposes, you can install Capact locally. To learn how to do that, see the [Development cluster](../development/development-guide.md#development-cluster) section in the "Development Guide" document.
+Use Capact CLI to create local Kubernetes cluster and install Capact on it.
+
+## Prerequisites
+- [Docker](https://www.docker.com/) installed
+- [`kubectl`](https://kubernetes.io/docs/tasks/tools/) installed
+- Capact CLI installed according to the [Getting Started](./../cli/getting-started.mdx#install) guide
+
+## Create local Kubernetes cluster
+
+To create local [kind](https://kind.sigs.k8s.io/) cluster, run the following command:
+
+```bash
+capact environment create kind
+```
+
+Verify that your local cluster is running by listing all Nodes:
+
+```bash
+kubectl get nodes
+```
+
+The output should be similar to:
+
+```bash
+NAME                            STATUS   ROLES    AGE    VERSION
+kind-dev-capact-control-plane   Ready    master   3m5s   v1.19.1
+```
+
+## Install Capact
+
+Once you have your local Kubernetes cluster set up, install Capact with the following command:
```

Comment:
```suggestion
Once you have your local Kubernetes cluster setup, install Capact with the following command:
```

Reply:
> Setup is a noun that refers to an arrangement. Set up is a phrasal verb that refers to making preparations for something.
In this context it's a verb, so I will keep it as it is.